src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl

```diff
@@ -265,34 +273,35 @@ function SplitExplicitSettings(FT::DataType=Float64;
 
     return SplitExplicitSettings(substeps,
                                  averaging_weights,
-                                 mass_flux_weights, 
-                                 Δτ, 
+                                 mass_flux_weights,
+                                 Δτ,
                                  timestepper)
 end
 
 # Convenience Functions for grabbing free surface
 free_surface(free_surface::SplitExplicitFreeSurface) = free_surface.η
 
-# extend 
+# extend
 @inline explicit_barotropic_pressure_x_gradient(i, j, k, grid, ::SplitExplicitFreeSurface) = zero(grid)
 @inline explicit_barotropic_pressure_y_gradient(i, j, k, grid, ::SplitExplicitFreeSurface) = zero(grid)
 
 # convenience functor
-function (sefs::SplitExplicitFreeSurface)(settings::SplitExplicitSettings)
-    return SplitExplicitFreeSurface(sefs.η, sefs.state, sefs.auxiliary, sefs.gravitational_acceleration, settings)
-end
+(sefs::SplitExplicitFreeSurface)(settings::SplitExplicitSettings) =
+    SplitExplicitFreeSurface(sefs.η, sefs.state, sefs.auxiliary, sefs.gravitational_acceleration, settings)
 
 Base.summary(sefs::SplitExplicitFreeSurface) = string("SplitExplicitFreeSurface with $(sefs.settings.substeps) steps")
-
 Base.show(io::IO, sefs::SplitExplicitFreeSurface) = print(io, "$(summary(sefs))\n")
 
 function reset!(sefs::SplitExplicitFreeSurface)
     for name in propertynames(sefs.state)
         var = getproperty(sefs.state, name)
         fill!(var, 0.0)
     end
+
     fill!(sefs.auxiliary.Gᵁ, 0)
     fill!(sefs.auxiliary.Gⱽ, 0)
+
+    return nothing
```

Comment:
I added this `return nothing` -- it had to be added, right? @glwagner?

Reply:
For sure, it's just a style issue but that is the style we have chosen.